vavr/src/test/java/io/vavr/PartialFunctionTest.java

```diff
@@ -0,0 +1,29 @@
+/*                        __    __  __  __    __  ___
+ *                       \  \  /  /    \  \  /  /  __/
+ *                        \  \/  /  /\  \  \/  /  /
+ *                         \____/__/  \__\____/__/.ɪᴏ
+ * ᶜᵒᵖʸʳᶦᵍʰᵗ ᵇʸ ᵛᵃᵛʳ ⁻ ˡᶦᶜᵉⁿˢᵉᵈ ᵘⁿᵈᵉʳ ᵗʰᵉ ᵃᵖᵃᶜʰᵉ ˡᶦᶜᵉⁿˢᵉ ᵛᵉʳˢᶦᵒⁿ ᵗʷᵒ ᵈᵒᵗ ᶻᵉʳᵒ
+ */
+package io.vavr;
+
+import io.vavr.collection.HashMap;
+import io.vavr.control.Option;
+import org.junit.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class PartialFunctionTest {
+
+    @Test
+    public void shouldReturnSome() {
+        Option<String> oneToOne = HashMap.of(1, "One").lift().apply(1);
+        assertThat(oneToOne instanceof Option.Some);
```

Comment:
I think this test is always green, because the assertion has no _check_ called.

Let's check the concrete value, i.e.

```java
assertThat(oneToOne).isEqualTo(Option.some("One"));
```